apps/web/models.py

```diff
@@ -33,11 +35,13 @@ class Team(models.Model):
 
     name = models.CharField(max_length=100)
     email = models.EmailField(max_length=70)
+    description = RichTextField(null=True)
```

Comment:
Angular can parse the `<i>` tags. So, there is no point in keeping this. Moreover, this is not a client side facing feature and hence we should not add extra unnecessary dependency.